packages/manager/modules/pci/src/projects/project/instances/stop/stop.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.instances.stop', {
+    url: '/stop?instanceId',
+    views: {
+      modal: {
+        component: 'pciInstancesInstanceStop',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      instanceId: /* @ngInject */ ($transition$) =>
+        $transition$.params().instanceId,
+      instance: /* @ngInject */ (
+        PciProjectsProjectInstanceService,
+        projectId,
+        instanceId,
+      ) => PciProjectsProjectInstanceService.get(projectId, instanceId),
+      goBack: /* @ngInject */ (goToInstances) => goToInstances,
+      breadcrumb: () => null,
```

Comment:
Relates to following module `@ovh-ux/ng-ui-router-breadcrumb`.
Isn't supposed to be defined as a dependency of following module: ovhManagerPciInstancesStop?

Reply:
Deps. has been added